iot-hub/Samples/service/AutomaticDeviceManagementSample/AutomaticDeviceManagementSample.cs

```diff
@@ -25,44 +25,44 @@ public AutomaticDeviceManagementSample(RegistryManager registryManager)
         public async Task RunSampleAsync()
         {
             Console.WriteLine("Create configurations");
-            await AddDeviceConfiguration("config001").ConfigureAwait(false);
-            await AddDeviceConfiguration("config002").ConfigureAwait(false);
-            await AddDeviceConfiguration("config003").ConfigureAwait(false);
-            await AddDeviceConfiguration("config004").ConfigureAwait(false);
-            await AddDeviceConfiguration("config005").ConfigureAwait(false);
+            await AddDeviceConfigurationAsync("config001");

```

Comment:
This still uses hard coded config names. Wasn't that the cause of our pipeline failures?

Did Nick have another PR which used dynamic names?